sdk/src/offchain_message.rs

```diff
@@ -2,13 +2,23 @@
 
 #![cfg(feature = "full")]
 
-use crate::{
-    hash::Hash,
-    pubkey::Pubkey,
-    sanitize::SanitizeError,
-    signature::{Signature, Signer},
+use {
+    crate::{
+        hash::Hash,
+        pubkey::Pubkey,
+        sanitize::SanitizeError,
+        signature::{Signature, Signer},
+    },
+    num_enum::{IntoPrimitive, TryFromPrimitive},
 };
 
+#[cfg(test)]
+static_assertions::const_assert_eq!(OffchainMessage::HEADER_LEN, 17);
+#[cfg(test)]
+static_assertions::const_assert_eq!(v0::OffchainMessage::MAX_LEN, 65515);
+#[cfg(test)]
+static_assertions::const_assert_eq!(v0::OffchainMessage::MAX_LEN_LEDGER, 1212);
+
```

Comment:
I think I'd rather these live by the const definitions to make them more useful for a reader.

Reply:
oh, I tried that first, I couldn't make it compile when these asserts are inside struct implementations. But I can move constants to the top level scope if you would prefer.

Reply:
Ah. Oh well, leave this as-is, then.